src/play/game/standard_game.ts

```diff
@@ -28,6 +28,12 @@ export class StandardGame extends Container {
   private storyboardOverlay: StoryboardLayerTimeline;
   private songProgressGraph: SongProgressGraph;
 
+  /**
+   * When audio ends it pauses itself and resets seek time to 0.
+   * There is no way to determine whether 
+   * audio has not yet begun or has already ended. 
+   */
+  private isStarted = false;
   private audio: Howl;
```

Comment:
This looks great! Would you mind adding an `isEnded` flag so that animations and videos don't freeze when the song ends?